lib/paths/local.go

```diff
@@ -229,6 +238,14 @@ func NewLocal(ctx context.Context, ls LocalStorage, index SectorIndex, urls []st
 
 		paths: map[storiface.ID]*path{},
 	}
+
+	expvar.Publish("localpath", expvar.Func(func() any {
```

Comment:
This is causing panic. We seem to be redeclaring this somewhere else.

Reply:
:thinking: weeeird, it works for me